src/kemal/cli.cr

```diff
@@ -3,13 +3,15 @@ require "option_parser"
 module Kemal
   # Handles all the initialization from the command line.
   class CLI
-    def initialize
+    def initialize(args)
       @ssl_enabled = false
       @key_file = ""
       @cert_file = ""
       @config = Kemal.config
       read_env
-      parse
+      if args != nil
+      	parse
```

Comment:
`parse` should actually receive `args` as argument and forward it to `OptionParser#parse`. Otherwise this whole thing doesn't make any more sense than a simple flag.

Reply:
thank you for your feedback @straight-shoota !